packages/sdk/src/index.ts

```diff
@@ -563,6 +566,18 @@ export {
   type AaveV3RedeemArgs,
 } from "./extensions/integrations/instances/aaveV3.js";
 
+// ./extensions/integrations/instances/compoundV2.js
```

Comment:
@KedziaPawel Note that you can generate this file using the exports script: https://github.com/enzymefinance/sdk/blob/main/scripts/exports.ts